### DISCUSSION TITLE
Citating Kratos (CITATION.cff files)

### DISCUSSION THREAD

**Discussion by jeyci94** [KratosMultiphysics/Kratos#12513] [General]
While searching what would be the proper way to cite a Kratos wiki page and generate the latex entry I came across the feature of GitHub supporting `CITATION.cff` files. These are basically plain text files which store citation information metadata.
GitHub has built-in support, when added to the repository it creates an APA or BibTeX style citation link that can be referenced. 
Something very nice is that it adds a "Cite this repository" entry to the Homepage under the About section providing a BibTeX snippet to simply copy. The feature is supported by GitHub, Zenodo, and Zotero.
I know the Homepage `README` has the section "_How to cite Kratos?_", just wanted to bring this to attention as an idea/suggestion, perhaps it makes it easier for people to cite Kratos, not only the articles but the repository itself. Maybe even for easier generation of the Zenodo record.
Some links in case it's of any interest or help!
[GitHub Blog: Support for citations](https://github.blog/2021-08-19-enhanced-support-citations-github/)
[CITATION.cff file](https://citation-file-format.github.io/)
[Generate a .cff file](https://citation-file-format.github.io/cff-initializer-javascript/#/)

**Comment by loumalouomega** [comment]
I did this 2 years ago and still blocked: https://github.com/KratosMultiphysics/Kratos/pull/10193
